doc/manual/src/language/constructs.md

```diff
@@ -217,19 +217,17 @@ three kinds of patterns:
     
     > **Warning**
     > 
-    > The `args@` expression is bound to the argument passed to the
-    > function which means that attributes with defaults that aren't
-    > explicitly specified in the function call won't cause an
-    > evaluation error, but won't exist in `args`.
-    > 
+    > `args@` binds the name `args` to the attribute set that is passed to the function.
+    > In particular, `args` does *not* include any default values specified with `?` in the function's set pattern.
+    >
```

Comment:
This is ok, but I think part of the problem is the original framing of `args` as matching the pattern at line 212.

```diff
-Here `args` is bound to the entire argument, which is further
+Here `args` is bound to the original argument, which is further
```

Perhaps we could also work with a single example. `args@{x, y ? 1, ...}` could replace all examples in the `@` pattern section.